pkgs/development/libraries/cre2/default.nix

```diff
@@ -17,7 +17,8 @@ stdenv.mkDerivation rec {
     libtool
     pkg-config
   ];
-  buildInputs = [ re2 texinfo ];
+  buildInputs = [ texinfo ];
+  propagatedBuildInputs = [ re2 ];
```

Comment:
personally, I'm not a fan of using `propagatedBuildInputs`, it doesn't play well with cross compilation and overrides.

Reply:
I don't think this is documented formally anywhere, but I've ran into issues with propagatedBuildInputs recently.